### PR TITLE
js_task-move-spider-DOM

### PR DESCRIPTION
**Pull request by AndriiYelieva**
- [DEMO LINK](https://AndriiYelieva.github.io/js_task-move-spider-DOM/)

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+    return;
+  }
+
+  const wallTop = wall.offsetTop;
+  const wallLeft = wall.offsetLeft;
+
+  const clickX = even.pageX - (wallLeft + spider.offsetWidth / 2);
+  const clickY = even.pageY - (wallTop + spider.offsetHeight / 2);
+
+  const widthBorder = 20;
```
Comment (IvanFesenko) [inline src/scripts/main.js:17]: Consider using a more descriptive name for 'widthBorder', such as 'wallBorderWidth'.

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,5 +1,27 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+document.addEventListener('click', (even) => {
```
Comment (IvanFesenko) [inline src/scripts/main.js:3]: use` event` or `e` cuz even has another meaning

File: src/scripts/main.js [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+  const spider = document.querySelector('.spider');
+
+  if (!wall) {
+    return;
+  }
+
+  const wallTop = wall.offsetTop;
+  const wallLeft = wall.offsetLeft;
+
+  const clickX = even.pageX - (wallLeft + spider.offsetWidth / 2);
```
Comment (IvanFesenko) [inline src/scripts/main.js:14]: Use 'event.clientX' and 'event.clientY' instead of 'event.pageX' and 'event.pageY' as per the task description.

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Good job. Lets make some improvement

**Review by mykhalenych — APPROVED** [review]

**Pull request closed without merging**